src/anemoi/utils/config.py

```diff
@@ -205,6 +205,23 @@ def load_any_dict_format(path) -> dict:
         if path.endswith(".toml"):
             with open(path, "rb") as f:
                 return tomllib.load(f)
+
+        if path == "-":
```

Comment:
This is a bit obscure. Should we be explicit here and use "/dev/stdin" instead of "-" ?

Reply:
This is the standard way in Unix.